templates/vars.yml.j2

```diff
@@ -1,5 +1,14 @@
 ---
+{% block vars scoped %}
```

Comment:
How does this change the behavior of the script?

Reply:
It makes the vars visible for the following includes, otherwise, I couldn't use them in the Nginx extension.